src/wp-includes/class-wp-dependencies.php

```diff
@@ -289,7 +289,6 @@ public function add_data( $handle, $key, $value ) {
 		if ( ! isset( $this->registered[ $handle ] ) ) {
 			return false;
 		}
-
```

Comment:
Generally, I think we can stick with the pattern of keep empty lines before returned values in this file.